src/renderer/component/threeViewer/internal/loader.js

```diff
@@ -0,0 +1,34 @@
+import { LoadingManager, STLLoader, OBJLoader } from './three';
+
+const Manager = ({ onLoad, onStart, onProgress, onError }) => {
+  const manager = new LoadingManager();
+  manager.onLoad = onLoad;
+  manager.onStart = onStart;
+  manager.onProgress = onProgress;
+  manager.onError = onError;
+
+  return manager;
+};
```

Comment:
I don't think this function is being used? I see it being called below, but that value isn't being used.

Reply:
https://github.com/lbryio/lbry-desktop/blob/34ba2cf3ebad8383e045d606ff8b5c27921e53a7/src/renderer/component/threeViewer/internal/loader.js#L24